packages/palette/src/elements/Carousel/Carousel.story.tsx

```diff
@@ -57,6 +57,22 @@ const Dynamic = () => {
   return <Demo widths={widths} />
 }
 
+const Navigation = () => {
+  const [initialCursor, resetCursor] = useState(0)
+  return (
+    <Box>
+      <Demo initialCursor={initialCursor} onChange={resetCursor} />
+      <Box display="flex" justifyContent="space-around">
+        <Clickable onClick={() => resetCursor(0)}>Navigate to page 1</Clickable>
```

Comment:
(nonblocking) - "Cursor" seems like somewhat technical niche vocabulary vs something like `initialPage` and `setPage` or `goToPage`